modules/collectionRegistrationMixin.js

```diff
@@ -357,42 +340,49 @@
        * @return {boolean} true if this collection fetches lazily.
        */
       collection.isLazyFetch = function() {
-        return collection.lazyFetch;
+        return this.lazyFetch;
       };
     };
 
     return {
       /**
-       * The super constructor / initialize method for collections.
+       * The constructor constructor / initialize method for collections.
        * Allocate new memory for the local references if they
        * were null when this method was called.
-       * @method super
+       * @param [options] {Object} - optional options object
+       * @param   [options.fetchHttpAction='POST'] {String} http action used to get objects by ids
+       * @param   [options.getByIdsUrl='/ids'] {String} path appended to collection.url to get objects by a list of ids
+       * @param   [options.lazyFetch=false] {Boolean} if set to true, the collection will assume that models do not change after first fetch
+       * @param   [options.fetchUsingTrackedIds=true] {Boolean} if set to false, cache.fetch() will not pass to fetchByIds with current tracked ids
+                                                               but will rather call the default fetch method.
+       * @method constructor
        */
-      super: function(args) {
-        args = args || {};
-        baseSuper.call(this, args);
-        this.isRequester = args && args.isRequester;
+      constructor: function(models, options) {
+        options = options || {};
+        base.call(this, models, options);
+        this.isRequester = options.isRequester;
+        this.parentInstance = options.parentInstance;
         if (!this.isRequester) {
-          cacheMixin(this, {
-            requestMap: {},
-            collectionTrackedIds: [],
-            knownPrivateCollections: {},
-            getByIdsUrl: args.getByIdsUrl
+          this.requestMap = {};
+          this.collectionTrackedIds = [];
+          this.knownPrivateCollections = {};
+          this.getByIdsUrl = options.getByIdsUrl || '/ids';
+          this.fetchHttpAction = 'POST';
+          this.lazyFetch = options.lazyFetch || false;
+          this.fetchUsingTrackedIds = options.fetchUsingTrackedIds !== false;
+          cacheMixin(this);
+        } else {
+          this.trackedIds = [];
+          this.listenTo(this.parentInstance, 'load-begin', function() {
+            this.trigger('cache-load-begin');
```

Comment:
done and done. "this" is correct, this is under test, the payload we can add if someone wants something.
the order goes: load-begin->cache-load-begin->FETCH->cache-load-complete->load-complete